core/feconf.py

```diff
@@ -1624,6 +1624,11 @@ def get_empty_ratings() -> Dict[str, int]:
 # The sort keys of submitted questions shown on the Contributor Dashboard.
 SUGGESTIONS_SORT_KEYS = [constants.SUGGESTIONS_SORT_KEY_DATE]
 
+# Auto-generated review message for an obsolete translation suggestion, i.e. a
+# suggestion corresponding to content that no longer exists.
+OBSOLETE_TRANSLATION_SUGGESTION_REVIEW_MSG = (
+    constants.OBSOLETE_TRANSLATION_SUGGESTION_REVIEW_MSG)
```

Comment:
Why is this needed why you cannot use `constants.OBSOLETE_TRANSLATION_SUGGESTION_REVIEW_MSG` directly.